info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:30:12.442Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:30:12.525Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:23:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:30:12.932Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:23:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:30:15.645Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:30:26.722Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:30:26.818Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:37:37.536Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:37:37.625Z"}
error: 
    status - 500 
    message - Error: User acess denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:33:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:37:43.203Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:29:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:38:01.145Z"}
error: 
    status - 500 
    message - Error: User acess denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:33:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:38:28.633Z"}
error: 
    status - 500 
    message - Error: User acess denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:33:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:38:31.571Z"}
error: 
    status - 500 
    message - Error: User acess denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:33:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:38:36.515Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:29:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:39:06.679Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:40:35.281Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:40:35.415Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:40:57.586Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:40:57.661Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:40:57.830Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:40:59.465Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:41:51.734Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:41:51.839Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:41:53.487Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:41:54.674Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:42:01.312Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:42:02.745Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:42:41.726Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:43:01.798Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:43:01.941Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:23)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:43:02.003Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:43:12.846Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:43:12.936Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:43:27.121Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:43:27.203Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:43:37.713Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:43:37.802Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:43:58.083Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:43:58.198Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:44:13.463Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:44:13.602Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:44:29.386Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:44:29.462Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:45:09.614Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:45:46.422Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:45:46.539Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:45:46.543Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:48:28.243Z"}
error: Could not connect to the database. connection timed out {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"Error: \n    at new MongooseError (D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\error\\mongooseError.js:7:1)\n    at new MongooseServerSelectionError (D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\error\\serverSelection.js:26:1)\n    at NativeConnection.Connection.openUri (D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Book_Store_App\\Book-Store\\node_modules\\mongoose\\lib\\index.js:1149:10)","timestamp":"2022-01-26T18:48:51.130Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:49:10.540Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:49:52.448Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:51:28.164Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:51:28.193Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:51:36.118Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:52:59.265Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:52:59.291Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:52:59.786Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:53:31.474Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:53:31.496Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:53:31.915Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:53:33.301Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:53:53.655Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:53:53.695Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:54:01.143Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:54:01.165Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:54:06.885Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:54:08.062Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:55:30.166Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:55:30.198Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:55:43.611Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:55:43.661Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:55:56.245Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:56:20.257Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:56:22.459Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:56:26.454Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:57:06.258Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:57:06.296Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:57:09.667Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T18:57:53.561Z"}
info: Connected to the database. {"timestamp":"2022-01-26T18:57:53.600Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:57:53.890Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T18:57:54.584Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:02:30.977Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:02:31.017Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:02:53.395Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:02:53.433Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:03:51.985Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:03:52.019Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:04:16.610Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:04:16.644Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:04:17.553Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:04:19.499Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:05:18.161Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:05:18.233Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:05:18.268Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:06:53.608Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:06:53.643Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:00.981Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:09.654Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:10.386Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:11.528Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:07:30.170Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:07:30.245Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:07:53.542Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:07:53.564Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:57.570Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:07:58.673Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T19:08:15.687Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:09:47.524Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:09:47.551Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:09:55.599Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:09:55.631Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:10:07.967Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:10:07.989Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:10:31.577Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:10:31.602Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:10:43.307Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:10:43.332Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:11:10.251Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:11:10.312Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:11:26.928Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:11:26.952Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:38:35.598Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:38:35.627Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:41:00.476Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:41:00.513Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:41:37.010Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:41:37.063Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:42:15.546Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:42:15.574Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:48:33.210Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:48:33.251Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:52:22.705Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:52:22.731Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:52:59.471Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:52:59.514Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:53:19.368Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:53:19.396Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:55:55.382Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:55:55.409Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:58:50.142Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:58:50.167Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:59:09.286Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:59:09.315Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:59:33.049Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T19:59:45.301Z"}
info: Connected to the database. {"timestamp":"2022-01-26T19:59:45.326Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:00:58.963Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:00:58.996Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:01:10.768Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:01:10.797Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:01:46.161Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:01:46.196Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:02:01.030Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:02:01.068Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:02:13.569Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:02:13.596Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:02:33.477Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:02:33.518Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:02:46.408Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:02:46.436Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:03:02.138Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:03:02.198Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:03:46.083Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:03:46.136Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:03:58.827Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:03:58.863Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:04:06.598Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:04:06.629Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:07:02.325Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:07:02.350Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:07:37.649Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:07:37.675Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:08:30.569Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:08:30.593Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:08:46.724Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:08:46.749Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:10:59.709Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:10:59.734Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:11:21.280Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:11:21.303Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:11:48.417Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:11:48.444Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:13:58.261Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:13:58.296Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:15:12.493Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:15:12.523Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:15:22.152Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:15:22.194Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:15:35.938Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:15:35.970Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:16:44.418Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:16:44.446Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:16:58.023Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:16:58.054Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:18:10.309Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:18:10.363Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:18:44.608Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:18:44.687Z"}
error: 
    status - 500 
    message - Error: Book not found
    at _callee8$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:153:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/FetchBook/61f16f81f572003d1c0fce8e 
    method - GET 
    IP - ::1
   {"timestamp":"2022-01-26T20:18:45.103Z"}
error: 
    status - 500 
    message - Error: Book not found
    at _callee8$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:153:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/FetchBook/61f16f81f572003d1c0fce8e 
    method - GET 
    IP - ::1
   {"timestamp":"2022-01-26T20:18:45.754Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:19:17.333Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:19:17.364Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:19:25.797Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:19:25.829Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:20:06.103Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:20:06.171Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:20:12.893Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:20:12.918Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:22:33.646Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:22:33.677Z"}
error: 
    status - 500 
    message - Error: Book not found
    at _callee6$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:137:9)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/UpdateBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-26T20:22:57.595Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-26T20:23:44.659Z"}
info: Connected to the database. {"timestamp":"2022-01-26T20:23:44.683Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:27:04.184Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:27:04.246Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:54:13.512Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:54:13.610Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:54:57.181Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:54:57.263Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:55:23.903Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:55:24.004Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:56:13.648Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:56:13.733Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:56:30.146Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:56:30.187Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:56:47.044Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:56:47.111Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:56:58.435Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:56:59.167Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:57:35.169Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:57:35.251Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:57:43.014Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:57:43.100Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:57:57.449Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:57:57.508Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:58:20.783Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:58:20.857Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T05:58:52.839Z"}
info: Connected to the database. {"timestamp":"2022-01-27T05:58:52.904Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at logger (D:\Book_Store_App\Book-Store\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:224:16)
    at done (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T05:58:54.728Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:00:05.909Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:00:05.942Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at logger (D:\Book_Store_App\Book-Store\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:224:16)
    at done (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:00:06.633Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:08:17.537Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:08:17.644Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:08:24.050Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:08:24.106Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:09:20.764Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:09:20.830Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt malformed
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:63:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T06:09:24.236Z"}
error: 
    status - 500 
    message - Error: Please enter corret mail id or password
    at _callee2$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:57:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/admin_login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T06:10:09.996Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:11:52.636Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:11:52.727Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:12:20.073Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:12:20.105Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:14:07.537Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:14:07.619Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:15:24.570Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:15:24.604Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:16:22.752Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:16:22.784Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:17:40.305Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:17:40.371Z"}
error: 
    status - 500 
    message - Error: User Already Exsist
    at _callee$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:22:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/user_registration 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:17:40.421Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/checkuser.middleware.js:15:10)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/user_login 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:17:40.601Z"}
error: 
    status - 500 
    message - Error: Mail id does not exsist
    at _callee3$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:80:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/forgetpassword 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:17:40.745Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee4$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:89:29)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.resetPassword (D:\Book_Store_App\Book-Store\src\services\/user.service.js:86:27)
    at _callee4$ (D:\Book_Store_App\Book-Store\src\controllers\/user.controller.js:55:36)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at resetPassword (D:\Book_Store_App\Book-Store\src\controllers\/user.controller.js:53:27)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at Function.handle (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:174:3)
    at router (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at logger (D:\Book_Store_App\Book-Store\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:323:13)
    at D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:341:12)
    at next (D:\Book_Store_App\Book-Store\node_modules\express\lib\router\index.js:275:10)
    at D:\Book_Store_App\Book-Store\node_modules\body-parser\lib\read.js:130:5
    at invokeCallback (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:224:16)
    at done (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (D:\Book_Store_App\Book-Store\node_modules\raw-body\index.js:273:7)
    at IncomingMessage.emit (events.js:387:35)
    at endReadableNT (internal/streams/readable.js:1317:12)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) 
    url - /api/v1/users/resetpassword 
    method - POST 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:17:40.782Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:18:05.996Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:18:06.031Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:27:33.581Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:27:33.778Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:28:07.827Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:28:07.865Z"}
error: 
    status - 500 
    message - Error: Book not found
    at _callee6$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:140:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/UpdateBook 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T06:28:08.711Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T06:55:17.152Z"}
info: Connected to the database. {"timestamp":"2022-01-27T06:55:17.381Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T06:55:22.888Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T06:55:36.462Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books/AddBook 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T06:55:37.863Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T07:09:30.330Z"}
info: Connected to the database. {"timestamp":"2022-01-27T07:09:30.416Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T07:12:14.139Z"}
info: Connected to the database. {"timestamp":"2022-01-27T07:12:14.210Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T07:16:48.518Z"}
info: Connected to the database. {"timestamp":"2022-01-27T07:16:48.590Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T07:18:36.998Z"}
info: Connected to the database. {"timestamp":"2022-01-27T07:18:37.041Z"}
error: 
    status - 500 
    message - Error: Book not found
    at _callee6$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:140:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/UpdateBook 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T07:18:37.853Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:41:29.910Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:41:29.961Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:43:34.449Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:43:34.487Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:49:36.287Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:49:36.323Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:49:50.707Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:49:50.754Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:51:08.187Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:51:08.219Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:51:35.708Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:51:35.769Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:58:28.307Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:58:28.361Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T08:59:40.910Z"}
info: Connected to the database. {"timestamp":"2022-01-27T08:59:40.943Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:03:40.410Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:03:40.502Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T09:04:01.310Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid token
    at Object.verify (D:\Book_Store_App\Book-Store\node_modules\jsonwebtoken\verify.js:75:17)
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:26:37)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12 
    url - /api/v1/books 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T09:04:03.176Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:09:00.749Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:09:00.888Z"}
error: 
    status - 500 
    message - ReferenceError: req is not defined
    at _callee8$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:163:10)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.fetchByID (D:\Book_Store_App\Book-Store\src\services\/user.service.js:161:23) 
    url - /api/v1/books/61f192f3dfe88d63f0c5caac 
    method - GET 
    IP - ::1
   {"timestamp":"2022-01-27T09:09:22.504Z"}
error: 
    status - 500 
    message - ReferenceError: req is not defined
    at _callee8$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:163:10)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.fetchByID (D:\Book_Store_App\Book-Store\src\services\/user.service.js:161:23) 
    url - /api/v1/books/61f17240a93a8152a82ece0e 
    method - GET 
    IP - ::1
   {"timestamp":"2022-01-27T09:09:42.474Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:10:28.774Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:10:28.846Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:10:40.884Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:10:40.962Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:11:45.165Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:11:45.260Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:11:55.051Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:11:55.145Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:12:23.637Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:12:23.721Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:12:46.576Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:12:46.675Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee6$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:130:15)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/61f17240a93a8152a82ece0e 
    method - PATCH 
    IP - ::1
   {"timestamp":"2022-01-27T09:13:20.790Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:14:15.389Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:14:15.490Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:14:33.290Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:14:33.372Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:32:50.428Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:32:50.487Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:33:06.833Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:33:06.875Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:34:31.660Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:34:31.695Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "BookID" (type string) at path "_id" for model "Book"
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/BookID 
    method - PATCH 
    IP - ::ffff:127.0.0.1
   {"timestamp":"2022-01-27T09:34:32.597Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:35:13.415Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:35:13.453Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:36:24.927Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:36:24.964Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:36:58.147Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:36:58.190Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:58:33.275Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:58:33.381Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:59:38.577Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:59:38.616Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T09:59:59.102Z"}
info: Connected to the database. {"timestamp":"2022-01-27T09:59:59.157Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:00:15.719Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:00:15.768Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:00:53.273Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:00:53.312Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:07:59.899Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:07:59.933Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:10:39.813Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:10:39.848Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:14:08.012Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:14:08.150Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:16:29.175Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:16:29.239Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:17:28.220Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:17:28.253Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:18:21.815Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:18:21.846Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:18:48.569Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:18:48.644Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:19:26.611Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:19:26.673Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:20:03.154Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:20:03.194Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:24:07.334Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:24:07.372Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:25:22.999Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:25:23.107Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:40:39.929Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:40:39.974Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/61f276a8b6303238b4fe8092 
      method - DELETE 
      IP - ::ffff:127.0.0.1
     {"timestamp":"2022-01-27T10:40:40.891Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:42:49.730Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:42:49.831Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:43:40.847Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:43:40.906Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:47:04.429Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:47:04.484Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:56:53.042Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:56:53.133Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T10:59:05.260Z"}
info: Connected to the database. {"timestamp":"2022-01-27T10:59:05.306Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:01:46.708Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:01:46.809Z"}
error: 
    status - 500 
    message - ReferenceError: userr is not defined
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:27:28)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:15:3 
    url - /api/v1/books/ 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T11:02:25.375Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:50:47.505Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:50:47.582Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:53:51.709Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:53:51.777Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:54:03.698Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:54:03.783Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:54:23.558Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:54:23.686Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:54:28.939Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:54:29.041Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:54:51.628Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:54:51.713Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:55:05.308Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:55:05.418Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:55:11.423Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:55:11.526Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:55:17.822Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:55:17.901Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:55:31.270Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:55:31.319Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:56:37.734Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:56:37.814Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:56:57.714Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:56:57.814Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:57:08.413Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:57:08.512Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:57:21.855Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:57:21.929Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:57:36.330Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:57:36.429Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:57:44.491Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:57:44.590Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:57:59.903Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:58:00.001Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:58:10.275Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:58:10.377Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:58:21.393Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:58:21.454Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:58:32.373Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:58:32.438Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:59:07.612Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:59:07.678Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T11:59:17.782Z"}
info: Connected to the database. {"timestamp":"2022-01-27T11:59:17.867Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:01:00.671Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:01:00.758Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:01:14.445Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:01:14.493Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:01:58.976Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:01:59.057Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:02:54.308Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:02:54.430Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:05:26.093Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:05:26.151Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:21:54.324Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:21:54.404Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:22:33.168Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:22:33.239Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:23:18.622Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:23:18.714Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:27:41.085Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:27:41.160Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:32:00.410Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:32:14.516Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:32:14.593Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:32:20.492Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:32:20.567Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:32:30.427Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:32:30.487Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:35:16.940Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:35:17.063Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:37:58.223Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:37:58.331Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:38:47.361Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:38:47.479Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:39:36.033Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:39:36.112Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:39:52.857Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:39:52.981Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/61f17240a93a8152a82ece0e 
      method - PATCH 
      IP - ::1
     {"timestamp":"2022-01-27T12:40:39.081Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:42:07.836Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:42:08.001Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:43:43.008Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:43:43.082Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/%7B_id%7D 
      method - PATCH 
      IP - ::1
     {"timestamp":"2022-01-27T12:43:54.249Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:44:21.423Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:44:21.542Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/%7B_id%7D 
      method - PATCH 
      IP - ::1
     {"timestamp":"2022-01-27T12:44:29.570Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:44:55.144Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:44:55.202Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:46:19.186Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:46:19.318Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:49:38.858Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:49:38.972Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:50:30.817Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:50:30.905Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:51:09.169Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:51:09.282Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:51:15.969Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:51:16.045Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:54:25.987Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:54:26.090Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/61f17240a93a8152a82ece0e 
      method - PATCH 
      IP - ::1
     {"timestamp":"2022-01-27T12:56:30.250Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:57:28.910Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:57:28.991Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T12:57:58.703Z"}
info: Connected to the database. {"timestamp":"2022-01-27T12:57:58.817Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:01:08.429Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:01:08.492Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:02:14.532Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:02:14.619Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:02:24.311Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:02:24.384Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:03:06.030Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:03:06.110Z"}
error: 
    status - 500 
    message - Error: User Already Exsist
    at _callee$ (D:\Book_Store_App\Book-Store\src\services\/user.service.js:22:11)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/users/user_registration 
    method - POST 
    IP - ::1
   {"timestamp":"2022-01-27T16:03:38.845Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:05:27.405Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:05:27.467Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:12:23.000Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:12:23.113Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:12:33.509Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:12:33.580Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:12:42.102Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:12:42.185Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:13:01.107Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:13:01.206Z"}
error: 
    status - 500 
    message - Error: User Acess Denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:14)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/61f17240a93a8152a82ece0e 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-01-27T16:13:24.120Z"}
error: 
    status - 500 
    message - Error: User Acess Denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:14)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/61f18fec167c2b4680fb56c4 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-01-27T16:13:53.358Z"}
error: 
    status - 500 
    message - Error: User Acess Denied
    at _callee$ (D:\Book_Store_App\Book-Store\src\middlewares\/auth.middleware.js:30:14)
    at tryCatch (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:294:22)
    at Generator.next (D:\Book_Store_App\Book-Store\node_modules\regenerator-runtime\runtime.js:119:21)
    at asyncGeneratorStep (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at D:\Book_Store_App\Book-Store\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>) 
    url - /api/v1/books/%7B_id%7D?_id=61f18fec167c2b4680fb56c4 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-01-27T16:14:26.300Z"}
error: 
    status - 500 
    message - Error: Cast to undefined failed for value undefined at path "undefined"
    at new MongooseError (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\error\mongooseError.js:7:1)
    at new CastError (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\error\cast.js:31:7)
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/%7B_id%7D?_id=61f174f2ce436f2844f6173d 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-01-27T16:15:23.887Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-01-27T16:16:09.944Z"}
info: Connected to the database. {"timestamp":"2022-01-27T16:16:10.021Z"}
error: 
    status - 500 
    message - Error: Cast to undefined failed for value undefined at path "undefined"
    at new MongooseError (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\error\mongooseError.js:7:1)
    at new CastError (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\error\cast.js:31:7)
    at model.Query.exec (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Book_Store_App\Book-Store\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5) 
    url - /api/v1/books/%7B_id%7D?_id=61f174f2ce436f2844f6173d 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-01-27T16:16:11.554Z"}
